superset/migrations/shared/migrate_viz/base.py

```
@@ -45,11 +63,11 @@ def _migrate(self) -> None:
 
         rv_data = {}
         for key, value in self.data.items():
-            if key in self.mapping_keys and self.mapping_keys[key] in rv_data:
+            if key in self.rename_keys and self.rename_keys[key] in rv_data:
                 raise ValueError("Duplicate key in target viz")
```

I'm not sure if we should raise an error here. Maybe just silently override?